submission/loglist_refresher.go

```diff
@@ -30,26 +30,29 @@ const (
 	httpClientTimeout = 10 * time.Second
 )
 
-// LogListRefresher regularly reads Log-list and emits notifications when updates/errors
-// observed.
-type LogListRefresher struct {
-	// updateMu limits LogListRefresher to a single Refresh() at a time.
+// LogListRefresher is interface for Log List updates watcher.
+type LogListRefresher interface {
+	Refresh() (*loglist.LogList, error)
+}
+
+// LogListRefresherImpl regularly reads Log-list and emits notifications when
+// updates/errors observed. Implements LogListRefresher interface.
```

Comment:
Seems odd to export a thing named `..Impl` -- callers shouldn't need to care about the implementation.  So maybe:
 - make this `logListRefresherImpl` (or just `logListRefresher`)
 - restore `NewLogListRefresher(string) LogListRefresher {}` (no need for `*` in return type now it's an interface) but have it internally return a `logListRefresherImpl` instance.

Reply:
Renamed and changed signatures per suggestion.